src/utils.js

```diff
@@ -0,0 +1,23 @@
+import * as path from 'node:path';
+import * as fs from 'node:fs/promises';
+import { fileURLToPath } from 'node:url';
+import { errorMessage } from './consts.js';
+export const errorHandler = (incomingError, targetError) => {
+    if (incomingError.code = targetError) {
```

Comment:
I think accidentally you did a mistake here. You need to replace the assignment operator '=' with the comparison operator '==='. In other case the "If" block will work for each thrown error. By careful next time)
P.S. I didn't deduct points for this mistake.

Reply:
@nikburachenok oh, what a stupid typo from my side, thank you! :)